### PR TITLE
WIP: Adding manual payments

### PR DESCRIPTION
**Pull request by aalrayes**

### CODE REVIEW

**Review by ahoshaiyan — APPROVED** [review]

**Comment by ahoshaiyan** [comment]
@Fghurayri can you please check if you have any input, if not please merge it and release a new version

File: lib/src/widgets/credit_card.dart [hunk 1]
```diff
@@ -40,12 +40,17 @@ class _CreditCardState extends State<CreditCard> {
...
+
   @override
   void initState() {
     super.initState();
     setState(() {
       _tokenizeCard = widget.config.creditCard?.saveCard ?? false;
     });
+    setState(() {
+      _manualPayment = widget.config.creditCard?.manual ?? false;
+    });
```
Comment (Fghurayri) [inline lib/src/widgets/credit_card.dart:53]: Can you please merge setting the state in one block?

```dart
    setState(() {
      _tokenizeCard = widget.config.creditCard?.saveCard ?? false;
      _manualPayment = widget.config.creditCard?.manual ?? false;
    });
```

File: lib/src/models/credit_card_config.dart [hunk 2]
```diff
@@ -2,6 +2,7 @@
 class CreditCardConfig {
   ///  An option to save (tokenize) the card after a successful payment.
   bool saveCard;
+  bool manual;
```
Comment (Fghurayri) [inline lib/src/models/credit_card_config.dart:5]: Can you please add a one-liner doc comment just like the `saveCard` variable above?

**Review by Fghurayri — CHANGES_REQUESTED** [review]
Awesome work! @aalrayes 🚀👏 

I have left small notes for you. Can you please also add the release related changes?

1. Update the version of this package to `2.0.2` in: 
  * `pubspec.yaml` file
  * `src/moyasar.dart` file
2. Update the CHANGELOG file with the details of this new release
3. Update the Readme code example to include the usage of this new feature

**Comment by Fghurayri** [comment]
Hey @ahoshaiyan 👋 Please note that I don't have permission to merge

File: lib/src/widgets/credit_card.dart (hunk 1 above)
Comment (aalrayes) [reply lib/src/widgets/credit_card.dart:53]: sure 👍🏻

File: lib/src/models/credit_card_config.dart (hunk 2 above)
Comment (aalrayes) [reply lib/src/models/credit_card_config.dart:5]: sure 👍🏻

**Pull request merged**